challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |i| i == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject { |i| i == "dog" }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.count
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.take 2
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.reject { |i| !%w(dog cat).include?(i) }
```

Comment:
Great advice @jaybobo! I agree. I've not used the grep method on the Enumerable class before. I will definitely add this to the toolbox! Thanks again!